beacon-chain/slasher/service.go

```diff
@@ -134,7 +135,6 @@ func (s *Service) run() {
 
 // Stop the slasher service.
 func (s *Service) Stop() error {
-	s.cancel()
```

Comment:
Can we revert this change? I'm worried about long running processes not terminating on shutdown.